sdgym/benchmark.py

```diff
@@ -779,6 +855,78 @@ def _create_instance_on_ec2(script_content):
     print(f'Job kicked off for SDGym on {instance_id}')  # noqa
 
 
+def _handle_deprecated_parameters(
+    output_filepath, detailed_results_folder, multi_processing_config, run_on_ec2
+):
+    """Handle deprecated parameters and issue warnings."""
+    parameters_to_deprecate = {
+        'output_filepath': output_filepath,
+        'detailed_results_folder': detailed_results_folder,
+        'multi_processing_config': multi_processing_config,
+        'run_on_ec2': run_on_ec2,
+    }
+    parameters = []
+    for name, value in parameters_to_deprecate.items():
+        if value is not None and value:
+            parameters.append(name)
+
+    if parameters:
+        parameters = "', '".join(sorted(parameters))
+        message = (
+            f"Parameters '{parameters}' are deprecated in the `benchmark_single_table` "
+            'function and will be removed in October 2025. '
+            'Please consider using `output_destination` instead.'
+        )
```

Comment:
Let me know if this warning message makes sense. I introduce `output_destination`, but not all the deprecated parameters relate to saving data.